boards/posix/native_posix/Kconfig.board

```diff
@@ -1,13 +1,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
-config BOARD_NATIVE_POSIX
-	bool "Native POSIX"
+config BOARD_NATIVE_POSIX_32BIT
+	bool "Native POSIX for 32-bit host"
 	depends on SOC_POSIX
```

Comment:
As this is common for both, would it make sense to move it to BOARD_NATIVE_POSIX?

Reply:
>>  +	bool "Native POSIX for 32-bit host"
>>  	depends on SOC_POSIX
>>
>> As this is common for both, would it make sense to move it to BOARD_NATIVE_POSIX?
>
>This is the items pulled into the board selection menu. I'm not sure how
>it could be done differently. Or maybe I misunderstood what you're
>asking.

(it was just a nit) I meant the `depends on SOC_POSIX` line. Could it not be moved into the common BOARD_NATIVE_POSIX config.